packages/react-native/ReactAndroid/src/main/java/com/facebook/react/internal/views/text/fragments/MapBufferTextFragmentList.kt

```diff
@@ -0,0 +1,14 @@
+package com.facebook.react.internal.views.text.fragments
+
+import com.facebook.react.common.mapbuffer.MapBuffer
+
+/**
+ * A list of text fragments backed by a [MapBuffer]
+ */
+class MapBufferTextFragmentList(private val mFragments: MapBuffer) : TextFragmentList {
```

Comment:
```suggestion
class MapBufferTextFragmentList(private val fragments: MapBuffer) : TextFragmentList {
```

Reply:
I was just following the existing RN conventions, but I assume that when switching to Kotlin they can change too

Reply:
...but it seems I wasn't consistent within this PR